### DISCUSSION TITLE
Seq never displays the dashboard on some browsers

### DISCUSSION THREAD

**Discussion by dougcameron-infotrack** [datalust/seq-tickets#2089] [Help]
I have a Chrome-based browser (Brave) that has always worked well with Seq.  Lately, when i go to the seq dashboard, it will finish loading, but won't display the dashboard, only the colorful background animation.  If I open it in a different browser then it works fine, even other Brave browsers.  Ctrl-F5 doesn't fix it either.  Both videos are using Brave, one works, the other doesn't.

Doesn't work
https://github.com/datalust/seq-tickets/assets/81374171/6f0b480b-12f1-4d1e-88e7-662d5b49d81a

Works
https://github.com/datalust/seq-tickets/assets/81374171/792c2c27-f31b-4f08-b745-1d593a53bc40

**Comment by nblumhardt** [comment]
Thanks for the heads-up, Doug. If you open Brave's developer tools console when the issue occurs, are any error messages printed in the console output, or any failed network requests shown? Thanks!

**Comment by dougcameron-infotrack** [comment]
Oh my. The act of opening developer tools has fixed the issue (can’t believe I didn’t try that). It’s been doing this for months! Several other have reported the same issue to me.  I’ll have them try opening the dev tools too

**Reply by nblumhardt** [reply]
That's odd - thanks for the follow-up, Doug. Could this be a Brave bug, then?

**Comment by dougcameron-infotrack** [comment]
Happens in other chrome browsers too
________________________________

**Comment by dougcameron-infotrack** [comment]
More followup.  We have many users and they all experience this behavior in a variety of browsers.  We even get a scenario where the Signal bar/column is missing and there's a red box that says something like could not load signals.  The fix is always to open F12 dev tools and do a hard refresh.  The "Disable cache" checkbox has to be checked during the refresh or it doesn't work.

Doug
________________________________

**Reply by liammclennan** [reply]
This is sounding like a network issue. Is there some sort of proxy or gateway between Seq and the browser?
